app/client/cli/debug.go

```diff
@@ -210,51 +190,16 @@ func sendDebugMessage(cmd *cobra.Command, debugMsg *messaging.DebugMessage) {
 	}
 
 	// if the message needs to be broadcast, it'll be handled by the business logic of the message handler
+	//
+	// DISCUSS_THIS_COMMIT: The statement above is false. Using `#Send()` will only
```

Comment:
Things have definitely changed/evolved so I'll provide some context & pointers, but would like to get your thoughts/ideas on what the right thing to do is.

For example, take this target to send a tx:
```Makefile
.PHONY: send_local_tx
send_local_tx: ## A hardcoded send tx to make LocalNet debugging easier
	go run -tags=debug app/client/*.go Account Send --non_interactive 00104055c00bed7c983a48aac7dc6335d7c607a7 00204737d2a165ebe4be3a7d5b0af905b0ea91d8 1000
```

- We sent it to one address
- That one address adds it to the mempool
- It needs to be gossiped through the network

In `utility/utility_message_handler.go`, we handle a `TxGossip` like so where `HandleTransaction` just adds it to the local mempool.

```go
func (u *utilityModule) HandleUtilityMessage(message *anypb.Any) error {
	switch message.MessageName() {
	case messaging.TxGossipMessageContentType:
		msg, err := codec.GetCodec().FromAny(message)
		// ...
		if txGossipMsg, ok := msg.(*typesUtil.TxGossipMessage); !ok {
			return fmt.Errorf("failed to cast message to UtilityMessage")
		} else if err := u.HandleTransaction(txGossipMsg.Tx); err != nil {
			return err
		}
		// ...
	default:
		return coreTypes.ErrUnknownMessageType(message.MessageName())
	}
	// ...
}
```

**Our goal is**: Gossip `TxGossip` throughout the network
**Question**: WHat do we change to make it happen? `Account Send`? `s/Send/Broadcast`? How we handle `case messaging.TxGossipMessageContentType`? Wdyt?